File: my-submissions/1528-kids-with-the-greatest-number-of-candies/solution.js
/**
 * @param {number[]} candies
 * @param {number} extraCandies
 * @return {boolean[]}
 */
var kidsWithCandies = function(candies, extraCandies) {
    let result = [];
    let maxVal = Math.max(...candies);

    for(let c of candies){
      result.push(c + extraCandies >= maxVal);
    }
    
    // console.log(result);
    return result;
};

File: my-submissions/1528-kids-with-the-greatest-number-of-candies/solution.java
class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {
        List<Boolean> result = new ArrayList<>();
        int max = 0;

        for(int c: candies){
           max = Math.max(c, max);
        }

        for(int candyCount : candies){
           result.add(candyCount + extraCandies >= max);
        }

        return result;
    }
}
